allenact_plugins/manipulathor_plugin/vibrations.csv

```diff
@@ -0,0 +1,2089 @@
+object,dist,scene
```

Comment:
I think we should host this CSV file somewhere, e.g. S3, and have it automatically downloaded when/if people need it. It's probably a good idea to also add some code that makes sure we're only using these vibration distances for a particular set of THOR commit ids as these distances will change as THOR is updated. Also: we probably don't need to store the vibration distances for object/scene pairs that have a distance of 0.0 (we can just assume that if a pair isn't in the dictionary then it must have 0 distance).

Reply:
I agree with Luca. Maybe we can release this with each stable release of the AI2THOR?

Reply:
Also, have in mind that the object IDs are going to be changed in the new release. So it might be good to release the script that generates this csv file.

Reply:
Now I temporarily move the csv to my repo. We can decide its location later.

Reply:
That sounds great to me Kiana. We'll just have to remember to do this :)